recipes/atk/all/test_package/CMakeLists.txt

```diff
@@ -1,7 +1,16 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
 project(test_package LANGUAGES C)
 
 find_package(atk REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
 target_link_libraries(${PROJECT_NAME} PRIVATE atk::atk)
+
+if (CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    find_package(glib CONFIG REQUIRED)
+    target_link_libraries(${PROJECT_NAME} PRIVATE glib::glib-2.0)
+else()
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(glib-2.0 REQUIRED IMPORTED_TARGET glib-2.0)
+    target_link_libraries(${PROJECT_NAME} PRIVATE PkgConfig::glib-2.0)
+endif()
```

Comment:
@SpaceIm Sorry for bothering you again.
The `CMakeDeps` generator for `glib` does not populate the `includedir` values at all for some reason, so I had to apply the same logic that you have previously used in `glib`'s test_package: https://github.com/conan-io/conan-center-index/blob/master/recipes/glib/all/test_package/CMakeLists.txt#L6-L17
Was the PkgConfig approach used there as a workaround or simply as a test of the PkgConfig generator? 
The need for this workaround here looks like a bug to me, though, and downgrading to older v2 Conan does not help either.